browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3860,6 +3866,18 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_ADD_NFTS" desc="How to import NFTs in the NFT grid.">
       Ready to add some? Just click the button below to import.
     </message>
+    <message name="IDS_BRAVE_WALLET_ENABLE_NFT_AUTODISCOVERY_MODAL_HEADER" desc="Enable NFT auto-discovery modal header">
+      Want your NFTs displayed automatically?
+    </message>
+    <message name="IDS_BRAVE_WALLET_ENABLE_NFT_AUTODISCOVERY_MODAL_DESCRIPTION" desc="Enable NFT auto-discovery modal description">
+      Brave Wallet can use a third-party service to automatically display your NFTs. Brave will share your wallet addresses with <ph name="BEGIN_LINK_1">&lt;LINK_1&gt;</ph>SimpleHash<ph name="END_LINK_1">&lt;/LINK_1&gt;</ph> to provide this service. <ph name="BEGIN_LINK_2">&lt;LINK_2&gt;</ph>Learn more.<ph name="END_LINK_2">&lt;/LINK_2&gt;</ph>
```

Comment:
Is there a better way than introducing these `LINK_1` and `LINK_2` tags? I see we use `Utils.createSpanForSurroundedPhrase` in other places or is it not flexible enough for this specific styling?

Reply:
[createSpanForSurroundedPhrase](https://github.com/brave/brave-core/blob/master/android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java#L1337) only works for one span in text not multiple.
I found the above solution more reliable due to specific tags and it's also being used in chromium code at many places.
- [Revamp view](https://source.chromium.org/chromium/chromium/src/+/main:chrome/android/java/src/org/chromium/chrome/browser/ntp/RevampedIncognitoDescriptionView.java;l=155-157?q=RevampedIncognitoDescriptionView) 
- [Privacy view](https://source.chromium.org/chromium/chromium/src/+/main:chrome/android/java/src/org/chromium/chrome/browser/privacy/settings/PrivacySettings.java;l=208-210?q=PrivacySettings)
Is there any issue with this approach?